### DISCUSSION TITLE
Unreal to verify

### DISCUSSION THREAD

**Discussion by oleshkus** [community/community#139031] [GitHub Education] [answered]
### Hi everyone,

My location or the location of my college is not being recognized correctly. I've already sent 3 similar applications with minor tweaks. Each time I get this.

You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
Lower i uploaded image that i'm doing photo of. Cause my native language is russian, i have to translate it to give possibility to understand what is written there. I don't know, what should i do next, to fix my issues. 
### Please guys, help me
![image](https://github.com/user-attachments/assets/f2aa54d5-fd55-48ae-ab24-76f8f0b6a7ec)

**Comment by alx-mp4** [comment, marked as answer]
Hi @oleshkus, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
